resources/css/utilities.css

```diff
@@ -1,3 +1,9 @@
+@layer components {
+  .container {
+    @apply sm:max-w-[1024px] xl:max-w-[1280px];
+  }
+}
```

Comment:
This fixes a bug where SM and MD no longer stretched the width of the page.

Reply:
Interesting! I missed that it was overwriting that all that time oO
What do you think of using the tailwind theme helper here?
To have those breakpoints defined only in one place:
```css
@apply sm:max-w-[theme('screens.lg')] xl:max-w-[theme('screens.xl')];
```

Reply:
That is better- I should've reached for this. I keep forgetting `theme()` is a thing.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1821/commits/829e3e1b556b1fbf8e8e6d55e2932d703e30a65c.